docs/DeveloperGuide.md

```diff
@@ -2,7 +2,7 @@
 
 ## Acknowledgements
```

Comment:
In the Ui UML Diagram, the accessibility of the attributes and methods are represented by different symbols (such as diamonds, square etc). Maybe you might want to disable the skinparam to display as '+' and '-' as how we usually do those?